setup-slave.sh

```diff
@@ -21,12 +21,15 @@ HOSTNAME=$PRIVATE_DNS  # Fix the bash built-in hostname variable too
 
 echo "Setting up slave on `hostname`..."
 
-# Work around for R3 instances without pre-formatted ext3 disks
+# Mount options to use for ext3 and xfs disks (the ephemeral disks
+# are ext3, but we use xfs for EBS volumes to format them faster)
```

Comment:
I know this comment is just copied from where it was before, but would you mind updating it to say "...xfs disks (we use xfs for EBS volumes to format them faster)" (since we don't currently use ext3 anywhere)